eth/tracers/tracer.go

```diff
@@ -238,6 +238,13 @@ func (dw *dbWrapper) pushObject(vm *duktape.Context) {
 		return 1
 	})
 	vm.PutPropString(obj, "exists")
+
+	// Push the wrapper for statedb.Empty
+	vm.PushGoFunction(func(ctx *duktape.Context) int {
+		ctx.PushBoolean(dw.db.Empty(common.BytesToAddress(popSlice(ctx))))
+		return 1
+	})
+	vm.PutPropString(obj, "isEmpty")
```

Comment:
I think `isEmpty` is a better name, but it looks like other wrapper use the snakeCased method name verbatim (ie `"empty"`). At most this is a nitpick...

Reply:
No strong preference. `isEmpty` feels to better serve its purpose. Changed to `empty` at 16fb0ec.